src/main/java/seedu/address/logic/commands/AttendanceCommand.java

```diff
@@ -0,0 +1,215 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ATTENDANCE_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DROP_OFF;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PICK_UP;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Scanner;
+
+import com.fasterxml.jackson.core.util.DefaultPrettyPrinter;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.pet.Name;
+import seedu.address.model.pet.Pet;
+import seedu.address.model.pet.Phone;
+import seedu.address.storage.JsonAdaptedAttendance;
+
+public class AttendanceCommand extends Command {
+
+    public static final String COMMAND_WORD = "attendance";
```

Comment:
As suggested at the UG section, I will suggest to shorten the COMMAND_WORD to `att`